lib/util/common.js

```diff
@@ -184,7 +184,9 @@ function resolveLocalRef (jsonSchema, externalSchemas) {
 
   // $ref is in the format: #/definitions/<resolved definition>/<optional fragment>
   const localRef = jsonSchema.$ref.split('/')[2]
-  return resolveLocalRef(externalSchemas[localRef], externalSchemas)
+  if (externalSchemas[localRef]) return resolveLocalRef(externalSchemas[localRef], externalSchemas)
+  // $ref is in the format: #/components/schemas/<resolved definition>
+  return resolveLocalRef(externalSchemas[jsonSchema.$ref.split('/')[3]], externalSchemas)
```

Comment:
We are splitting `localRef` on line 186, could we use that split instead?

Reply:
Of course, I'll do it ASAP